runelite-client/src/main/java/net/runelite/client/plugins/chatfilter/ChatFilterConfig.java

```diff
@@ -63,4 +64,37 @@ default String filteredRegex()
 	{
 		return "";
 	}
+
+	@ConfigItem(
+		keyName = "filterYourself",
+		name = "Filter Yourself",
+		description = "Filter your own messages",
+		position = 4
+	)
+	default boolean filterYourself()
+	{
+		return true;
+	}
+
+	@ConfigItem(
+		keyName = "filterFriends",
+		name = "Filter Friends",
+		description = "Filter your friends' messages",
+		position = 5
+	)
+	default boolean filterFriends()
+	{
+		return true;
```

Comment:
Default both this and the clan one to false.

Reply:
done